project-management/src/main/java/life/qbic/projectmanagement/application/communication/Attachment.java

```diff
@@ -0,0 +1,12 @@
+package life.qbic.projectmanagement.application.communication;
+
+/**
+ * <b>MailAttachment</b>
+ *
+ * <p>Encapsulates information about an attachment, e.g. a file added to an email</p>
+ *
+ * @since 1.0.0
+ */
+public record Attachment(String content, String name) {
```

Comment:
It probably needs encoding/type information. Like for example `plain/text` or similar.

Reply:
Tbh, I am not sure if the attachment implementation should stay like it is. However this is not the scope of the refactor, I am focusing on layering it out properly.